Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/UserPropActivity.java

```diff
@@ -172,6 +178,54 @@ void showUser() {
         subscribeInterceptdesk.setChecked((user.uLocalSubscriptions & Subscription.SUBSCRIBE_INTERCEPT_DESKTOP) != 0);
         subscribeInterceptmedia.setChecked((user.uLocalSubscriptions & Subscription.SUBSCRIBE_INTERCEPT_MEDIAFILE) != 0);
 
+        Channel chan = new Channel();
+        ttclient.getChannel(user.nChannelID, chan);
+
+        boolean hasVoice = true;
+        for (int i = 0; i < chan.transmitUsers.length; i++) {
```

Comment:
Put these loops into the Utils class so you're not repeating the same code. The method should have as input `userid`, `transmitUsers` and `StreamType`.